src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx

```diff
@@ -73,7 +73,7 @@ export default function VideoVisitLocation({ appointment, facility }) {
             )}
           </>
         )}
-        {kind === VIDEO_TYPES.gfe && (
+        {(kind === VIDEO_TYPES.gfe || isAtlas) && (
```

Comment:
I think it would make sense for the providers to be below the location info, especially for ATLAS appointments since there's explanatory text at the top that talks about the facility below it.

Reply:
Ok makes sense

Reply:
<img width="669" alt="image" src="https://user-images.githubusercontent.com/8315447/120816146-7d7d8180-c51e-11eb-99be-42c2430a13a9.png">